bin/aap-docs-bccutil.sh

```diff
@@ -0,0 +1,26 @@
+# Since master.adoc and docinfo.xml are not in the aap-docs repo, you can't use bccutil / ccutil to build aap-docs.
+# This function fetches files from Gitlab, runs bccutil, and deletes the added files.
+# 
+# Add this function to your ~/.zshrc or ~/.bashrc file, then run source ~/.bashrc or source ~/.zshrc
+#
+# Prerequisites:
+# You must be connected to the VPN to run this function, as it fetches files from Gitlab.
+# You must have set up a git remote called "gitlab" for the Gitlab production repo. 
+#
+# Procedure:
+# Run this function from a title directory, for example aap-on-aws
+# To run this function, type aap-docs-build <title-directory>, for example:
+# aap-docs-build aap-on-azure
+# aap-docs-build aap-on-aws
+# aap-docs-build aap-on-gcp
+#
+    function aap-docs-build () {
+    git fetch gitlab aap-clouds-latest
+    printf "*** Building $1 docs ***\n"
+    git show gitlab/aap-clouds-latest:$1/master.adoc > master.adoc
+    git show gitlab/aap-clouds-latest:$1/docinfo.xml > docinfo.xml
+    git show gitlab/aap-clouds-latest:$1/title-attributes.adoc > title-attributes.adoc
+    # In the following line, substitute the command that you use to build the docs with ccutil / bccutil. The example here is for a mac. Delete the hashtag before the command.
```

Comment:
At what point? If it's the runCCutil bit, maybe have it as <runCCutil>?

Reply:
@ianf77 What command do you use to run ccutil?
I'm on a mac, so I use "leviathin"

Reply:
At the level of the document, so where I'd find master.adoc I'd run bccutil compile --lang en-US --format html-single --open, though I now have an alias as that's a bit long-winded, so now I just use docb.

Reply:
Thanks: I'll add in your full command as an example for RHEL.

Reply:
@ianf77 
I added example commands for mac and RHEL.
Does this section make sense now?